test/unit/behaviors.spec.js

```diff
@@ -16,390 +24,480 @@ describe('Behaviors', function() {
+          behaviors: [behaviors.foo, behaviors.bar, {
+            behaviorClass: behaviors.baz
           }]
         });
 
-        this.view = new this.View();
+        fooView = new FooView();
       });
 
       it('should instantiate behaviors passed directly as a class', function() {
-        expect(this.behaviors.foo).to.have.been.calledOnce;
-        expect(this.behaviors.bar).to.have.been.calledOnce;
+        expect(behaviors.foo).to.have.been.calledOnce;
+        expect(behaviors.bar).to.have.been.calledOnce;
       });
 
       it('should instantiate behaviors passed with behaviorClass', function() {
-        expect(this.behaviors.baz).to.have.been.calledOnce;
+        expect(behaviors.baz).to.have.been.calledOnce;
       });
     });
+
+    afterEach(function() {
+      Marionette.Behavior.restore();
+      Marionette.Behaviors.behaviorsLookup.restore();
+    });
   });
 
   describe('behavior initialize', function() {
+    let behaviorOptions;
+    let initializeStub;
+    let behaviors;
+    let FooView;
+    let fooView;
+
     beforeEach(function() {
-      this.behaviorOptions = {foo: 'bar'};
-      this.initializeStub = this.sinon.stub();
+      behaviorOptions = {foo: 'bar'};
+      initializeStub = this.sinon.stub();
 
-      this.behaviors = {
-        foo: Marionette.Behavior.extend({initialize: this.initializeStub})
+      behaviors = {
+        foo: Behavior.extend({initialize: initializeStub})
       };
-      Marionette.Behaviors.behaviorsLookup = this.behaviors;
 
-      this.View = Marionette.View.extend({
-        behaviors: {foo: this.behaviorOptions}
+      this.sinon.stub(Marionette.Behaviors, 'behaviorsLookup', function() {
+        return behaviors;
+      });
+
+      FooView = View.extend({
+        behaviors: {foo: behaviorOptions}
       });
-      this.view = new this.View();
+      fooView = new FooView();
     });
 
     it('should have a cidPrefix', function() {
-      var fooBehavior = new this.behaviors.foo();
+      var fooBehavior = new behaviors.foo();
       expect(fooBehavior.cidPrefix).to.equal('mnb');
     });
 
     it('should have a cid', function() {
-      var fooBehavior = new this.behaviors.foo();
+      var fooBehavior = new behaviors.foo();
 
       expect(fooBehavior.cid).to.exist;
     });
 
     it('should call initialize when a behavior is created', function() {
-      expect(this.initializeStub).to.have.been.calledOnce;
+      expect(initializeStub).to.have.been.calledOnce;
     });
 
     it('should call initialize when a behavior is created', function() {
-      expect(this.initializeStub).to.have.been.calledOnce.and.calledWith(this.behaviorOptions, this.view);
+      expect(initializeStub).to.have.been.calledOnce.and.calledWith(behaviorOptions, fooView);
     });
 
-    it('should set _behaviors', function() {
-      expect(this.view._behaviors.length).to.be.equal(1);
+    afterEach(function() {
+      Marionette.Behaviors.behaviorsLookup.restore();
```

Comment:
Thanks, but looking at it right now I think it's not need :) because of 
https://github.com/marionettejs/backbone.marionette/blob/master/test/setup/setup.js#L48
https://github.com/marionettejs/backbone.marionette/blob/master/test/setup/setup.js#L52